scripts/install

```diff
@@ -84,7 +84,11 @@ fi
 # Build cjdns
 if ! [ -x "/opt/cjdns/cjdroute" ]; then
     here=`pwd`
-    cd /opt/cjdns && sudo Seccomp_NO=1 NO_NEON=$CJDNS_NO_NEON CFLAGS="$CJDNS_CFLAGS" ./do && cd $here
+    if [[ $RPI_REVISION == *"00"* ]]; then
```

Comment:
Can we switch up in https://github.com/Famicoman/prototype-cjdns-pi2/blob/05c76b3eb07a3fc5ae7227c296dfe4045fce1689/scripts/install#L16 where we do all other Pi revision-specific things?